generators/spring-service/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
+    Creates a new JHipster service: this is a simple transactional Spring service bean.
+
+Example:
+    jhipster spring-service Foo
```

Comment:
I am not sure that this command will work. 

It should be something like `jhipster --blueprint generator-jhipster-kotlin spring-service Foo`

@deepu105  can you confirm the above

Reply:
It doesn't work:

![screen shot 2018-02-25 at 19 46 11](https://user-images.githubusercontent.com/5684688/36645128-99021bae-1a64-11e8-86bf-2fa2821bdedc.png)

That's what I was trying to ask in #8 , it looked a bit confusing on how to test a newly added generator.